lua/starfall/permissions/providers_sh/url_whitelist.lua

```diff
@@ -180,3 +167,15 @@ simple [[i.redd.it]]
 --- you don't even wanna know
 
 simple [[static1.e621.net]]
+
+-- ipfs
+--- Examples:
+--- https://ipfs.io/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/I/m/Ellis_Sigil.jpg
+
+simple [[ipfs.io]]
+
+-- neocities
+--- Examples:
+--- https://fauux.neocities.org/LainDressSlow.gif
+
+pattern [[[A-Za-z0-9](?:[A-Za-z0-9%-]{0,61}[A-Za-z0-9])?%.neocities%.org/(.+)]
```

Comment:
pattern [[([%w-_]+)%.neocities%.org/(.+)]]
Seems a lot better, also you forgot ] at the end

Reply:
Will that defend against blah.neocities.org.mywebsite.com?

Reply:
Ah just noticed the `/` maybe that will prevent that

Reply:
It's modified discord pattern, I tested them all so I am sure its fine :)

Reply:
All right, nice.